### ISSUE TITLE
Runtime error on model.fit(...): Incompatible shapes: [64,59,93] vs. [1,93,1]

### ISSUE THREAD

**Issue by juanramonua** [keras-team/keras#8029]
python 3.6.2, keras 2.0.8, tensorflow 1.3.0

Runtime error on model.fit(...):

InvalidArgumentError (see above for traceback): Incompatible shapes: [64,59,93] vs. [1,93,1]
	 [[Node: training/RMSprop/gradients/dense_1/add_grad/BroadcastGradientArgs = BroadcastGradientArgs[T=DT_INT32, _class=["loc:@dense_1/add"], _device="/job:localhost/replica:0/task:0/cpu:0"](training/RMSprop/gradients/dense_1/add_grad/Shape, training/RMSprop/gradients/dense_1/add_grad/Shape_1)]]

**Comment by JeniaNovellusDx** [comment]
try changing the input shape to (x,y, channels) and also provide your code... We don't know what is "..."

**Comment by juanramonua** [comment]
Indeed, your suggestion is correct and the program works now.

I suggest to insert the bellow lines in de code:

from keras import backend as K
K.set_image_data_format('channels_last')

Thanks.

**Comment by harshini-gadige** [comment]
This issue is resolved by changing the input shape to (x,y, channels) as suggested above. Hence closing the issue.
